scania-digital-design-system/sdds
# feat(side menu): collapse button improved & header updated in demos

**Describe pull-request**  
- improvement side menu collapse button in Angular demos(sticky position on the bottom of the screen)
- update header(v2.0.0) in all Angular demos and React demo 

**Solving issue**  

Fixes: [AB#1371](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1371)

**How to test**  

1. Go to Angular(11, 12,13) demo and React demo
2. Check header and side menu collapse button the look and behaviour
